scripts/test/hyriseBenchmarkJoinOrder_test.py

```diff
@@ -29,6 +28,10 @@ def main():
     arguments["--clients"] = "1"
     arguments["--scheduler"] = "false"
 
+    # Binary tables would be written into the table_path. In CI, this path is shared by different targets that are
+    # potentially executed concurrently. This sometimes led to issues with corrupted binary files.
```

Comment:
This problem should occur for all benchmarks that cache binary tables, right? The other benchmark script files don't contain such a comment, so sad.

Reply:
Not an issue for the TPC-* benchmarks, which place the cached files in the current directory. You are right about the file-based benchmark though.